topsort-api-v2.yml

```diff
@@ -17,8 +17,8 @@ info:
   version: 2.0.0
 
 servers:
-  - url: https://api.topsort.com/v2
-    description: Auctions v2 path
+  - url: https://api.topsort.com/
```

Comment:
should this end in a slash? it didn't before.

Reply:
not sure what data type this is but I can remove it to be sure.